crates/grit-util/src/ranges.rs

```diff
@@ -245,11 +270,11 @@ impl MatchRanges {
 pub struct VariableMatch {
     pub name: String,
     pub scoped_name: String,
-    pub ranges: Vec<Range>,
+    pub ranges: Vec<ByteRange>,
```

Comment:
This is part of the API, so it will need to be a Range (with lines and columns).

Reply:
Ah, that's why the snapshots changed :) I knew something leaked through, I just hadn't spotted where. I guess I'll just give the `api` module its own `ApiVariableMatch` type uses `Range` still. Because this type is used internally as well, and that would trigger the on-demand computations at various places inside.

Reply:
This is going to take a bit more work. In `api.rs: Match::file_to_match` we have the matches, but we no longer have access to the source of the patterns in which the variable definitions exist...

Reply:
That's for the pattern file right? Realistically you might need to attach those at some point earlier in the process (ex. right after compilation).

Reply:
Think I got it fixed.

Reply:
Yep, snapshots look clean to me. Thanks!